hadolint.properties

```diff
@@ -0,0 +1,14 @@
+category=External Analyzers
+description=Sonar Hadolint Plugin allows users to import results from Hadolint into SonarQube.
+homepageUrl=https://github.com/cnescatlab/sonar-hadolint-plugin
+archivedVersions=
+publicVersions=1.0.0
+
+defaults.mavenGroupId=fr.cnes.sonarqube.plugins
+defaults.mavenArtifactId=sonar-hadolint-plugin
+
+1.0.0.description=Add ability to import Hadolint results, a Dockerfile language with highlightings & metrics, and a Quality Profile with Hadolint & Shellcheck rules.
+1.0.0.date=2020-10-22
+1.0.0.sqVersions=[7.9,8.5]
```

Comment:
This range should end with `LATEST` so we don't have to go through the PR process once 8.6 comes out in early Dec.